packages/make-styles-webpack-loader/tsconfig.json

```diff
@@ -1,17 +1,17 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["src"],
+  "include": ["__fixtures__/**/code.ts", "src"],
```

Comment:
can we use just with `__fixtures__/**` ?

Reply:
Due stripping/missing types in `output.ts` in some cases there were TS errors (it was in `babel-plugin`). If I will use `ts-ignore` or `ts-expect-output` results will not match 😕